assets/app/features/upgrade/controllers/upgrade-backup.controller.js

```diff
@@ -68,18 +75,9 @@ function() {
                             // Determine the content type from the header
                             contentType = headers['content-type'],
 
-                            backupBlob = new Blob([response.data], {type: contentType}),
-                            backupObjectUrl = URL.createObjectURL(backupBlob),
+                            backupBlob = new Blob([response.data], {type: contentType});
```

Comment:
I thougth that to trigger the download of zip files and binary files in general you were forced to set the `arraybuffer` responseType?